app/models/user.rb

```diff
@@ -1,6 +1,12 @@
 class User < ActiveRecord::Base
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable
+	has_many :pieces
+	has_many :games, :through => :pieces
```

Comment:
I'm not sure if this is the ideal way to do this (though I could be wrong). If you try Game.first.users with this setup, you get 32 user objects returned by the query, since there are 32 pieces that each have a user. Granted you could write some custom method to only list unique users (or they're may be a default method already in Rails), but that seems like unnecessarily adding complexity. Two other options could be to have a "GameInvitation" model with a player1 and player2 and game field to use the :through relationship for games and user (since there will only be 1 invitation per game), OR you could use "has_and_belongs_to_many" to link games and users directly. Let me know what you think.